simapp/app.go

```diff
@@ -276,52 +281,67 @@ func NewSimApp(
 	bApp.SetParamStore(app.ConsensusParamsKeeper.ParamsStore)
 
 	// add keepers
-	app.AccountKeeper = authkeeper.NewAccountKeeper(appCodec, runtime.NewKVStoreService(keys[authtypes.StoreKey]), authtypes.ProtoBaseAccount, maccPerms, authcodec.NewBech32Codec(sdk.Bech32MainPrefix), sdk.Bech32MainPrefix, authtypes.NewModuleAddress(govtypes.ModuleName).String())
+
+	app.AuthKeeper = authkeeper.NewAccountKeeper(appCodec, runtime.NewKVStoreService(keys[authtypes.StoreKey]), authtypes.ProtoBaseAccount, maccPerms, authcodec.NewBech32Codec(sdk.Bech32MainPrefix), sdk.Bech32MainPrefix, authtypes.NewModuleAddress(govtypes.ModuleName).String())
+
+	accountsKeeper, err := accounts.NewKeeper(
+		runtime.NewKVStoreService(keys[accounts.StoreKey]),
+		app.AuthKeeper.AddressCodec(),
+		appCodec.InterfaceRegistry().SigningContext(),
+		app.MsgServiceRouter(),
+		app.GRPCQueryRouter(),
+		accountstd.AddAccount("counter", counter.NewAccount),
```

Comment:
this is what we do to write an account, we give it a name and pass in an account creator function in this case counter.NewAccount

Reply:
would it be better to have the creation of the keeper and the adding of accounts as two separate steps?

Reply:
This would only be needed if say you have `keeperA` depending on `accountsKeeper` and then `accountA` depending on `keeperA`, this is a pattern that I think we want to remove from the SDK.